docs/src/main/asciidoc/migrations.adoc

```diff
@@ -4,33 +4,38 @@
 
 == Migrations
 
-In the following document we will write about necessary migration steps between versions.
+This section covers from one version of Spring Cloud Contract Verifier to the next
```

Comment:
This section covers *upgrading* / *migrating* ?

Reply:
Thank you. I was apparently typing faster than I was thinking. I'll fix it.